docker/production/.env

```diff
@@ -16,6 +16,12 @@ INVENTREE_WEB_PORT=1337
 INVENTREE_DEBUG=False
 INVENTREE_LOG_LEVEL=WARNING
 
+# InvenTree admin account details
+# Un-comment (and complete) these lines to auto-create an admin acount
```

Comment:
Leaving unused passwords in the env can be dangerous - should we maybe also add a note about this?

Suggestion:
```
Remove the entered credentials after the first run
```

Reply:
Will do.